pigpen-cascading/src/main/java/pigpen/cascading/OperationUtil.java

```diff
@@ -13,6 +13,8 @@
 
 public class OperationUtil {
 
+  public static final String SENTINEL_VALUE = "39c63d213f5cba3c7";
```

Comment:
Interesting choice for a sentinel value. I generally use namespace qualified keywords - would a similar approach not work here?